installer/dev/main.cpp

```diff
@@ -31,6 +31,19 @@ int wmain(int argc, wchar_t *argv[])
         }
     }
 
-    return ProjectReunionInstaller::DeployPackages(quiet);
+    const HRESULT deployPackagesResult = ProjectReunionInstaller::DeployPackages(quiet);
+    if (!quiet)
+    {
+        if (SUCCEEDED(deployPackagesResult))
+        {
+            std::cout << "All packages were installed successfully." << std::endl;
+        }
+        else
+        {
+            std::wcerr << "One or more packages failed to install. Result: 0x" << std::hex << deployPackagesResult << std::endl;
```

Comment:
Does `deployPackagesResult` include the package(s) that failed? If not that would make for a nifty future enhancement so the error could report '%d %s failed to install (...packages...)", count, pluralize(count, "Package", "Packages")) or some such

Reply:
I want the DeployPackages to return a result object that contains detailed status so we can report those details, which packages failed, what was the failure for each package, etc.  For now since it fails the entire operation the first time a fatal error is encountered it returns that error. Each package is announced in output so it is fairly obvious which package failed currently, unless the failure occurred in obtaining the package information. For that the result object could at least include what the resource identifier is. But all of that was more churn than I wanted for this PR.